base_layer/core/src/chain_storage/postgres/migrations/2020-04-07-14-49_tx_outputs/up.sql

```diff
@@ -0,0 +1,15 @@
+create table tx_outputs(
+    hash TEXT NOT NULL PRIMARY KEY,
+    features_flags SMALLINT NOT NULL,
+    features_maturity BIGINT NOT NULL,
+    commitment TEXT NOT NULL,
+    proof BYTEA NULL,
```

Comment:
proofs of work could possibly be in a separate table with the UTXO hash as the foreign key..

Reply:
Could be but than you need another index for this. Its going to be a large field, but I am open to both options

Reply:
This will already be stored separately using postgres's TOAST system (https://giswiki.hsr.ch/PostgreSQL_-_Binary_Large_Objects)

Reply:
> PostgreSQL also supports a storage system called "TOAST" (The Oversized-Attribute Storage Technique) that automatically stores values larger than a single database page (typically 8 KB) 

PoW data is ±800bytes